node/toolrunner.ts

```diff
@@ -263,8 +266,16 @@ export class ToolRunner extends events.EventEmitter {
             // https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options
 
             //start the child process for both tools
-            var cpFirst = child.spawn(toolPathFirst, this.args, {cwd: ops.cwd, env: ops.env});
-            cp = child.spawn(toolPath, this.pipeOutputToTool.args, {cwd: ops.cwd, env: ops.env});
+            var spawnOptionsFirst = { } as child.SpawnOptions;
+            spawnOptionsFirst.cwd = ops.cwd;
+            spawnOptionsFirst.env = ops.env;
+            spawnOptionsFirst['windowsVerbatimArguments'] = ops.windowsVerbatimArguments;
```

Comment:
should we only set this if it's true?  might not matter.  since undocumented - if node ever removed it and validated unknown options, then only tasks that used this minor case option would be affected instead of all.  The flip side of the argument is we would find out sooner after adopting a new node.